libs/wingc/src/lifting.rs

```diff
@@ -197,6 +197,20 @@ impl<'a> Visit<'a> for LiftVisitor<'a> {
 				return;
 			}
 
+			// Inflight expressions that evaluate to a preflight type are currently unsupported because
+			// we can't determine exactly which preflight object is being accessed and therefore can't
+			// qualify the original lift expression.
+			if expr_phase == Phase::Inflight && expr_type.is_preflight_class() && v.ctx.current_property().is_some() {
+				report_diagnostic(Diagnostic {
+					message: format!(
+						"Reference to lifted preflight object of type \"{expr_type}\" of unknown origin, can't qualify the lift (see https://github.com/winglang/wing/issues/76 for details)"
```

Comment:
I'm a little worried this error message might be difficult to understand. Is there any way to make this clearer? "unknown origin" seems cryptic

Reply:
Here's another go:

Expression of type {T} references an unknown lift (unknown preflight object), can't qualify the lift (see...)

Is this any better?

Reply:
Sounds better. I'm wondering if we should define a term like "capabilities" in our docs to collectively describe all of these permissions and environment variables and such that are granted to inflight hosts by preflight objects. So your error message could be tweaked to:

`Expression of type {T} references an unknown preflight object, can't qualify its capabilities`

Or another alternative:

`Unable to determine the capabilities needed to use preflight object of type {T} inflight`

Reply:
Makes sense to use "capabilities", for now I'll take the `Expression of type...` suggestion above. Hope that's good enough.